README.md

```diff
@@ -48,6 +49,7 @@ These inputs are provided by our container actions.
 | `jenkinsfile` | String | Jenkinsfile | The relative path to Jenkinsfile. You can check [the official manual about Jenkinsfile](https://www.jenkins.io/doc/book/pipeline/syntax/). |
 | `pluginstxt` | String | plugins.txt | The relative path to plugins list file. You can check [the valid plugin input format](https://github.com/jenkinsci/plugin-installation-manager-tool#plugin-input-format). You can also refer to the [plugins.txt](plugins.txt) in this repository. |
 | `jcasc` | String | N/A | The relative path to Jenkins Configuration as Code YAML file. You can refer to the [demos](https://github.com/jenkinsci/configuration-as-code-plugin/tree/master/demos) provided by `configuration-as-code-plugin` and learn how to configure the Jenkins instance without using UI page. |
+| `initHook` | String | N/A | The relative path to the [Groovy init hook file](https://www.jenkins.io/doc/book/managing/groovy-hook-scripts/). |
```

Comment:
Instead of allowing just one init hook file, wouldn't it be better if the user provides an entire directory that becomes `groovy.init.d` in the container?

Reply:
I changed this option now. Users are required to provide a direcotry which contains groovy scripts.